### PR TITLE
Emitter,  particle accelerator bullet fix

### PR DESCRIPTION
**Pull request by ThatDan123**
Should fix emitter and particle accelerator bullet not showing up for clients, ~just build testing atm.~

This doesnt fix other bullets being invisible for shooter

### CODE REVIEW

**Comment by ThatDan123** [comment]
Tested and it works, wooo

**Comment by Bod9001** [comment]
is this going to conflict with autumn's work?

**Comment by ThatDan123** [comment]
nope

File: UnityProject/Assets/Scripts/Messages/Server/ShootMessage.cs [hunk 1]
```diff
@@ -115,7 +115,7 @@ public class CastProjectileMessage : ServerMessage
 		/// <summary>
 		/// Projectile being shot
 		/// </summary>
-		public Guid ProjectilePrefab;
+		public string ProjectilePrefabName;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Messages/Server/ShootMessage.cs:118]: I feel like this is a more flimsy reference

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Messages/Server/ShootMessage.cs:118]: its the same way the guns shoot their projectiles

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Messages/Server/ShootMessage.cs:118]: ok cool

**Pull request merged**